mongodb/connection_test.go

```diff
@@ -157,14 +157,29 @@ func TestSuite(t *testing.T) {
 			})
 
 			Convey("UpdateId should update data if document exists", func() {
-				_, err := conn.Collection(collection).UpdateById(context.Background(), 3, bson.M{"$set": bson.M{"new_key": 7892}})
+				_, err := conn.Collection(collection).UpdateById(context.Background(), 3, bson.M{"$set": bson.M{"new_key": 7892, "state": "second"}})
 				So(err, ShouldBeNil)
 
 				res := TestModel{}
 
 				err = queryMongo(conn, collection, bson.M{"_id": 3}, &res)
 				So(err, ShouldBeNil)
 				So(res.NewKey, ShouldEqual, 7892)
+				So(res.State, ShouldEqual, "second")
+
+				Convey("UpdateMany should update multiple matching documents", func() {
```

Comment:
Is this meant to be an inner convey for the `UpdateId`? Should it go up a level?